ERCS/erc-5564.md

```diff
@@ -4,7 +4,8 @@ title: Stealth Addresses
 description: Private, non-interactive transfers and interactions
```

Comment:
What is a "non-interactive interaction"?

Reply:
I've changed it to `engagements`. 

The `non-interactive` part is where one can transfer funds to a randomly generated stealth address based on the receiver's `stealthMetaAddress`, without the recipient having to take any action.

The additional `interactions` or now `engagements` can be thought of as other actions that can be done with the randomly generated stealth address beyond regular transfers. Think messaging, voting, etc.